extensions/npm/main.ts

```diff
@@ -171,18 +177,46 @@ if (!output.success) {
 async function checkDryRun(subcommand: string, args: string[]) {
   console.log(`[${green("phylum")}] Updating lockfile…`);
 
-  const cmd = new Deno.Command("npm", {
-    args: [
-      subcommand,
-      "--package-lock-only",
-      "--ignore-scripts",
-      ...args,
-    ],
-    stdout: "inherit",
-    stderr: "inherit",
-    stdin: "inherit",
-  });
-  const status = await cmd.spawn().status;
+  // Ensure lockfile is up to date.
+  let status;
+  if (
+    "install".startsWith(subcommand) ||
+    "isntall".startsWith(subcommand) ||
+    "add".startsWith(subcommand) ||
+    "update".startsWith(subcommand) ||
+    "udpate".startsWith(subcommand) ||
+    "upgrade".startsWith(subcommand)
+  ) {
+    // Run the command without installation if a new package is installed.
+    const cmd = new Deno.Command("npm", {
+      args: [
+        subcommand,
+        "--package-lock-only",
+        "--ignore-scripts",
+        ...args,
+      ],
+      stdout: "inherit",
+      stderr: "inherit",
+      stdin: "inherit",
+    });
+    status = await cmd.spawn().status;
+  } else {
+    // Run just install if no new package is added.
+    //
+    // This is necessary since `update` does not have the `package-lock-only`
```

Comment:
Got the fix backwards

```suggestion
    // This is necessary since `remove` does not have the `package-lock-only`
```